refactoring/signature-request/README.md

```diff
@@ -0,0 +1,137 @@
+# Refactoring - Signature Request Pages
```

Comment:
can this go into the docs folder intsead of a new top level folder?

Reply:
Good suggestion @brad-decker , I moved it 🙏